runbotics/runbotics-orchestrator-ui/src/pages/blog/post/[slug].tsx

```diff
@@ -27,27 +21,11 @@ interface Params extends Record<string, string> {
     slug: string;
 }
 
-export const getServerSideProps: GetServerSideProps<Props, Params> = async ({
-    res,
-    params,
-    locale,
-}) => {
-    let post: BlogPost | undefined;
-
-    if (isCacheUpToDate((locale ? locale : 'en') as Language)) {
-        post = getSinglePostCache(locale as Language, params.slug);
-        res.setHeader('X-Cache', 'HIT');
-    } else {
-        recreateCache(locale as Language);
-    }
+export const getServerSideProps: GetServerSideProps<Props, Params> = async ({ params, locale }) => {
```

Comment:
What about ServerSideProps instead of GetServerSideProps?

Reply:
this is interface imported from `'next'`